java-demo/src/main/java/org/pingcap/ticdc/cdc/KafkaMessage.java

```diff
@@ -0,0 +1,57 @@
+package org.pingcap.ticdc.cdc;
```

Comment:
```suggestion
package com.pingcap.ticdc.cdc;
```
Probably should use our real domain name?

Reply:
OK, I will fix it.